### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by mr-islam** [sufone/qawl-svelte#40] [General]
As salamu alaykum (peace be upon you)! 👋🏽

This is a space to discuss Qawl, share ideas and feedback, talk about your experiences and more 🤗

Things that we decide to do as a community can be moved to the Issues section and get worked on 👍🏽

**Comment by RangoMcRoony** [comment]
Alhmadulillah, this is a beautiful thing you are doing.

May your efforts always remain sincere and pure and may your intentions be seen by one whose mercy encompasses all things.

**Reply by mr-islam** [reply]
Ameen! Wa iyyakum ❤️ ❤️ !!

Thank you for your support, may Allah help us to do good deeds sincerely, and accept them - ameen!

**Reply by mr-islam** [reply]
Also, I just wanted to share https://layl.app – this is another app by Sufone, and of course it's open source. I've been working on Layl recently alhamdulillah, to support many languages. (This is the testing version: https://i18n.layl.app)

InshaAllah you will find Layl beneficial 😃 

Please do share both Qawl and Layl with family and friends and spread good, if you like!

**Reply by RangoMcRoony** [reply]
I'll check it out and I will definitely share it

**Comment by 64git** [comment]
Asalamu Alaykum, 
I like this program because I can view the Quran in a zoomed view making it easier to read and focus on the harakat, however, the usability of zooming should be improved as well as navigating around a page. If you can enable zooming through the mouse scroll and grabbing and moving around the page via mouse, this will improve usability. Whenever I zoom to much it will block the versus I need to see therefore, I need a way to move around! Additionally,  mouse scrolling should zoom where the cursor is at to make it easier! what are your thoughts on these new adjustments?
